File: assignment-11/qustion4.c
#include <stdio.h>

int CountFour(int iNo){
    int iDigit=0;
    int iCounter=0;
    
    while(iNo>0){
        iDigit=iNo%10;
        if(iDigit==4){
            iCounter++;
        }
         iNo=iNo/10;
    }
    return iCounter;
}
int main()
{
    int iValue=0;
    int iRet=0;

    printf("Enter number :\n");
    scanf("%d",&iValue);

    iRet=CountFour(iValue);
    printf("%d",iRet);

    return 0;
}

File: assignment-39/qustion4.java
import java.util.*;

class qustion4
{
    public static int ToggleBit(int iNo,int iPos){
    int iMask=0x00000001;
    int iResult=0;
    
    iMask=iMask<<(iPos-1);

        iResult=iMask ^ iNo;
        
        return iResult;
        
 }
    public static void main(String args[])
    {
        int iNo=0,iPos=0;
        int iRet=0;
        Scanner sobj=new Scanner(System.in);

        System.out.println("Enter number: ");
        iNo=sobj.nextInt();

        System.out.println("Enter position:");
        iPos=sobj.nextInt();

        iRet=ToggleBit(iNo, iPos);

        System.out.println(iRet);

    }
}